test/scripts/e2e.sh

```diff
@@ -124,6 +126,12 @@ duration "serial client runners"
 
 deactivate
 
+fi # if $SKIP_E2E_SUBS = ""
+
+if [ "${E2E_SUBS_ONLY}" != "" ]; then
```

Comment:
Choose a `SKIP` or `ENABLE` pattern, not both. I think `ENABLE_E2E_SUBS`, `ENABLE_E2E_GO` and `ENABLE_E2E_EXPECT` would be easier for someone new to these scripts to understand, `SKIP_*` might be easier to integrate.